src/GeositeFramework/js/Export.js

```diff
@@ -41,6 +41,19 @@ require(['use!Geosite'],
                 return 'css/print-portrait.css';
             }
         }
+        
+        function canSetPageLayout() {
```

Comment:
That may be a simpler route, I can test it out and see how it goes. I'll also add a blurb as a comment paraphrasing what I mentioned earlier.